assets/js/shareIntent.js

```diff
@@ -59,10 +63,12 @@ export function bindSharePopup(config) {
    */
   var handleIntent = (e) => {
     e = e || window.event;
-    var target = e.target || e.srcElement;
-    var m;
-    var left;
-    var top;
+    const winHeight = window.screen.height;
+    const winWidth = window.screen.width;
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).